guides/source/working_with_javascript_in_rails.md

```diff
@@ -343,39 +347,6 @@ This generates a form with:
 <input data-disable-with="Saving..." type="submit">
 ```
 
-Dealing with Ajax events
-------------------------
-
-Here are the different events that are fired when you deal with elements
-that have a `data-remote` attribute:
-
-NOTE: All handlers bound to these events are always passed the event object as the
-first argument. The table below describes the extra parameters passed after the
-event argument. For example, if the extra parameters are listed as `xhr, settings`,
-then to access them, you would define your handler with `function(event, xhr, settings)`.
-
-| Event name          | Extra parameters | Fired                                                       |
-|---------------------|------------------|-------------------------------------------------------------|
-| `ajax:before`       |                  | Before the whole ajax business, aborts if stopped.          |
-| `ajax:beforeSend`   | xhr, options     | Before the request is sent, aborts if stopped.              |
-| `ajax:send`         | xhr              | When the request is sent.                                   |
-| `ajax:success`      | xhr, status, err | After completion, if the response was a success.            |
-| `ajax:error`        | xhr, status, err | After completion, if the response was an error.             |
-| `ajax:complete`     | xhr, status      | After the request has been completed, no matter the outcome.|
-| `ajax:aborted:file` | elements         | If there are non-blank file inputs, aborts if stopped.      |
-
```

Comment:
Why was this whole section removed?

Reply:
Because it is repeated below with the correct extra parameters here:
https://github.com/dimroc/rails/blob/master/guides/source/working_with_javascript_in_rails.md#rails-ujs-event-handlers

Reply:
Another, more accurate table, exists below. Screenshot:

![image](https://user-images.githubusercontent.com/635121/31862308-9bb83702-b709-11e7-890a-6728cfe3d1c5.png)

Reply:
Ok!! Maybe we should add a link to old version of table for jquery-ujs users?

http://guides.rubyonrails.org/v5.1/working_with_javascript_in_rails.html#dealing-with-ajax-events

Reply:
Sure. How does having a note mentioning the old jquery-ujs with a to jquery-ujs wiki sound? https://github.com/rails/jquery-ujs/wiki/ajax

Reply:
Yes, lets change to that. That used to(is still) the official old events doc.